### PR TITLE
janitoring: Small changes

### PR DESCRIPTION
**Pull request by vargac**
* Added some stuff to .gitignore (executables and swap files)
* Removed unused variable from arduino test/example

Signed-off-by: vargac <simon.varga123@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,4 +16,11 @@
 *.out
 *.app
 example
-./shell
+/vic
+tests/test-vic-exec
```
Comment (mrshu) [inline .gitignore]: @vargac can we replace these lines with `tests/test-vic-*`?

**Comment by mrshu** [comment]
@vargac Looks good, just one small nitpick!

File: .gitignore [hunk 2]
```diff
@@ -16,4 +16,11 @@
 *.out
 *.app
 example
-./shell
+/vic
```
Comment (mrshu) [inline .gitignore]: @vargac what does this actually capture? `/vic` might mean many things...

File: .gitignore (hunk 1 above)
Comment (vargac) [inline .gitignore]: @mrshu it would match also tests/test-vic-exec.c, what we don't want to.

File: .gitignore (hunk 2 above)
Comment (vargac) [inline .gitignore]: @mrshu it is result from 'make shell'

File: .gitignore (hunk 1 above)
Comment (mrshu) [inline .gitignore]: @vargac thanks!

File: .gitignore (hunk 2 above)
Comment (mrshu) [inline .gitignore]: @vargac well, didn't you mean `./vic` then? I am not exactly sure but `/vic` matches, well, `/vic`.

Did it actually work? Could you see `vic` binary in `git status`?

Comment (vargac) [inline .gitignore]: @mrshu 'A leading slash matches the beginning of the pathname.' It works like a charm

Comment (mrshu) [inline .gitignore]: @vargac Thanks, sorry for not reading the docs =)

**Comment by mrshu** [comment]
Thanks @vargac, looks good to me!

**Pull request merged**